hypertrace-gradle-docker-java-application-plugin/src/main/resources/application-start-script.template.sh

```diff
@@ -28,4 +28,11 @@ FIPS_CLASSPATH="\${FIPS_CLASSPATH%:}"
 # If FIPS_CLASSPATH is not empty, prepend it to classpath with colon as delim
 CLASSPATH="\${FIPS_CLASSPATH:+\${FIPS_CLASSPATH}:}/app/resources:/app/classes:/app/localLibs/*:/app/orgLibs/*:/app/externalLibs/*"
 echo "Using the classpath \$CLASSPATH"
+
+# Check for trust store password file
+if [ ! -z "\$TRUST_STORE_PWD_FILE" ] && [ -f "\$TRUST_STORE_PWD_FILE" ]; then
```

Comment:
can we do it using environment variable as well? secret can be mounted both as file or env variable.